src/vcpkg/commands.build.cpp

```diff
@@ -1162,17 +1162,15 @@ namespace vcpkg
         std::vector<Path> files;
         std::vector<std::string> hashes;
 
-        size_t i = 0;
-        for (auto& filestr : abi_info.pre_build_info->hash_additional_files)
+        for (size_t i = 0; i < abi_info.pre_build_info->hash_additional_files.size(); ++i)
         {
-            Path file(filestr);
+            auto& file = abi_info.pre_build_info->hash_additional_files[i];
             if (file.is_relative() || !fs.is_regular_file(file))
             {
                 Checks::msg_exit_with_message(VCPKG_LINE_INFO, msgInvalidValueHashAdditionalFiles, msg::path = file);
             }
-            const auto hash =
-                vcpkg::Hash::get_file_hash(fs, file, Hash::Algorithm::Sha256).value_or_exit(VCPKG_LINE_INFO);
-            abi_tag_entries.emplace_back(fmt::format("additional_file_{}", i++), hash);
+            auto hash = Hash::get_file_hash(fs, file, Hash::Algorithm::Sha256).value_or_exit(VCPKG_LINE_INFO);
+            abi_tag_entries.emplace_back(fmt::format("additional_file_{}", i), std::move(hash));
```

Comment:
why is this a two line change instead of a single line? I see what you are doing here

Reply:
by removing const and vcpkg this fits into one line